mmdet/models/dense_heads/deformable_detr_head.py

```diff
@@ -96,63 +89,12 @@ def init_weights(self) -> None:
             for m in self.reg_branches:
                 nn.init.constant_(m[-1].bias.data[2:], 0.0)
 
-    def forward(self, x: Tuple[Tensor],
-                batch_img_metas: List[dict]) -> Tuple[Tensor, ...]:
+    def forward(self, hs: Tensor, init_reference,
```

Comment:
@Li-Qingyun we should rename this `hs` to a more reasonable name

Reply:
hidden_states?